csm/conf/etc/csm/csm.conf

```diff
@@ -133,6 +133,7 @@ SUPPORT_BUNDLE:
     symlink_path: "/tmp/support_bundle/"
     cluster_file_path : "/opt/seagate/cortx/provisioner/pillar/components/cluster.sls"
     ssh_user : "root"
+    alerts_file_path: "/tmp/fru_alerts.json"
```

Comment:
Changes this Path to the path recieved in command this will allow the temporary file being deleted once bundle is generated.

Reply:
implemented review comments